core/modules/user/user.module

```diff
@@ -3228,7 +3402,32 @@ function user_form_field_ui_field_edit_form_submit($form, &$form_state) {
 function user_register_page() {
   // We need to set the page title because this menu entry is a local task.
   backdrop_set_title(t('Create new account'));
-  return backdrop_get_form('user_register_form');
+  $user_login_appearance = config_get('system.core', 'user_login_appearance');
```

Comment:
All three page callbacks rely solely on the config item, which means, they bypass any hook implementation of hook_user_login_paths_alter().
That affects user_register_page(), user_login_page() and user_pass_page() as far as I can see.